### PR TITLE
example for dumping file

### PR DESCRIPTION
**Pull request by vasiliyb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ Options:
   --silent=false                Set to true to disable all non-error output
   --no-progress=false           Set to true to disable the progress bar
 ```
+#### Example
+
+```
+dyndump dump --filename="tableOut" --s3-bucket="myS3BucketName" --s3-prefix="/" myTableName
```
Comment (gwatts) [inline README.md]: Thanks for the contribution - I agree some examples would be good

This one could probably use a bit of work though - Using `/` as an `--s3-prefix` option would seem unusual; usually you want some indication of the  name of the table you're backing up, or other metadata (per the example for that option)

This example would also cause the dumped table to be stored both on file locally, and in S3 - I imagine most people want to do one or the other - Perhaps examples for both would be better

Comment (vasiliyb) [reply README.md]: created a new branch with examples locally, can you plz give me write access to create a branch? ` git push --set-upstream origin vasiliy/examples` fails :)

Comment (vasiliyb) [reply README.md]: was able to make changes via the github UI to this branch!  ptal

Comment (gwatts) [reply README.md]: thanks - Merged!

**Pull request merged**